### PR TITLE
remove tableName from QueryBuilderArgs

### PR DESCRIPTION
**Pull request by ra1861**
A follow-up to #135 to remove unnecessary internal property _table_name_.

### CODE REVIEW

**Comment by ra1861** [comment]
@labianchin

File: dbeam-core/src/main/java/com/spotify/dbeam/avro/BeamJdbcAvroSchema.java [hunk 1]
```diff
@@ -55,8 +55,7 @@ public static Schema createSchema(Pipeline pipeline, JdbcExportArgs args, Connec
     final long elapsedTimeSchema = (System.nanoTime() - startTime) / 1000000;
     LOGGER.info("Elapsed time to schema {} seconds", elapsedTimeSchema / 1000.0);
 
-    JobNameConfiguration.configureJobName(
-        pipeline.getOptions(), dbName, args.queryBuilderArgs().tableName());
+    JobNameConfiguration.configureJobName(pipeline.getOptions(), dbName);
```
Comment (labianchin) [inline dbeam-core/src/main/java/com/spotify/dbeam/avro/BeamJdbcAvroSchema.java]: Can we keep the table name (when available) as part of the `jobName`? Often the exported table is a 1-1 mapping to the output endpoint. I guess we can take `tableName` from `args`..

Comment (ra1861) [reply dbeam-core/src/main/java/com/spotify/dbeam/avro/BeamJdbcAvroSchema.java]: Done.

File: dbeam-core/src/test/java/com/spotify/dbeam/avro/BeamJdbcAvroSchemaTest.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  public void verifyTableNameOk() {
+    String expected = "abba";
+    DBeamPipelineOptions args = Mockito.mock(DBeamPipelineOptions.class);
+    when(args.getTable()).thenReturn(expected);
+
+    String actual = BeamJdbcAvroSchema.getTablename(args);
+
+    Assert.assertEquals(expected, actual);
+  }
+}
```
Comment (labianchin) [inline dbeam-core/src/test/java/com/spotify/dbeam/avro/BeamJdbcAvroSchemaTest.java]: I don't think we need these tests with Mockito. I hope this is covered via some other tests..

File: dbeam-core/src/main/java/com/spotify/dbeam/avro/BeamJdbcAvroSchema.java [hunk 3]
```diff
@@ -72,6 +72,16 @@ public static Schema createSchema(Pipeline pipeline, JdbcExportArgs args, Connec
     return generatedSchema;
   }
 
+  static String getTablename(PipelineOptions args) {
```
Comment (labianchin) [inline dbeam-core/src/main/java/com/spotify/dbeam/avro/BeamJdbcAvroSchema.java]: Or simply something like:

```
String tableName = pipelineOptions.as(DBeamPipelineOptions.class).getTable();
if (tableName != null) {
  JobNameConfiguration.configureJobName(pipeline.getOptions(), dbName, tableName);
} else {
  JobNameConfiguration.configureJobName(pipeline.getOptions(), dbName);
}
```

**Pull request merged**